modules/ROOT/pages/installation/osx.adoc

```diff
@@ -19,6 +19,8 @@ Select the appropriate tar.gz distribution for your platform.
 . Extract the contents of the archive, using `tar -xf` <filename>.
 For example, `tar -xf neo4j-community-{neo4j-version-exact}-unix.tar.gz`.
 . Place the extracted files in a permanent home on your server and set the environment variable `NEO4J_HOME` to point to the extracted directory, for example, `export NEO4J_HOME=/path/to/_<NEO4J_HOME>` to make it easier to refer to it later.
+. xref:configuration/file-locations.adoc#file-locations-file-locations[Change the default locations] of the _data_, _conf_, _certificates_, _licenses_, and _plugins_ (if you plan to use custom plugins) directories by setting the environment variable `NEO4J_CONF` and the respective xref:configuration/configuration-settings.adoc#_server_directories_settings[`server.directories.*`] settings to point to the desired locations.
+Storing your Neo4j files outside the installation can simplify the upgrade process later.
```

Comment:
1. can we commit to saying that it WILL simplify the upgrade process?
2. Earlier the extract directory is referred to as `<NEO4J_HOME>` but here it is `outside the installation`. I think "Storing your Neo4j files outside `NEO4J_HOME`" might be more consistent. This is a minor point though, I leave the wording up to you.
3. This is quite an important point, maybe it should be in a callout for greater visibility?
Kind of like:

INFO
**Storing your Neo4j files outside `NEO4J_HOME` will simplify the upgrade process later.**
Otherwise after upgrading Neo4j, the data will still be stored under the old installation folder and is at risk of being accidentally deleted if an uninstall is attempted.

Reply:
About the note, how about:
```
[NOTE]
====
Storing your Neo4j files outside `NEO4J_HOME` will simplify the upgrade process later because you will be able to replace the DBMS binaries without affecting the configuration and state.
Otherwise, these Neo4j files will remain in the old installation folder and may be accidentally overwritten during an upgrade or deleted during a subsequent uninstall. 
====
```

Reply:
@renetapopova great!